app/forms/select_email_form.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class SelectEmailForm
+  include ActiveModel::Model
+  include ActionView::Helpers::TranslationHelper
+
+  attr_reader :user, :selected_email
+
+  validate :owns_selected_email
+
+  def initialize(user)
+    @user = user
+  end
+
+  def owns_selected_email
```

Comment:
Minor:

- Typically, validation methods are prefixed "validate_"
- Since we're separating public/private APIs in this class already, this feels like it's a private method, not part of the form's public API